samples/unit-tests/navigator/navigator/demo.js

```diff
@@ -1144,3 +1144,58 @@ QUnit.test(
         );
     }
 );
+
+QUnit.test('Navigator dafault dataLabels enabled, #13847.', function (assert) {
+    const chart = Highcharts.stockChart('container', {
+        plotOptions: {
```

Comment:
If you have only one series there is no need to declare the `plotOptions`. Simply add that config in the `series`.